jasmine/spec/inverted-index-test.js

```diff
@@ -52,7 +53,7 @@ describe('Inverted Index class test suite', function () {
 
       it('Should not have any books indexed yet', function () {
         let newInvIndex = new InvertedIndex();
```

Comment:
Why recreating another instance of `InvertedIndex` since you have `invIndex = new InvertedIndex();` in your `beforeEach()`